.tx/config

```diff
@@ -0,0 +1,9 @@
+[main]
+host = https://app.transifex.com
+
+[o:foreman:p:foreman:r:hammer-cli-foreman-admin]
```

Comment:
For consistency I'm thinking of renaming this to use underscores. https://github.com/theforeman/foreman-plugin-overview/pull/7/files#r1265350759

Reply:
Updating domain here will make impossible to automatically pull and merge the translations. We need to update the domain here and in Transifex. I'm not sure what's he exact procedure, but I'd guess that we update the domain here, re-create the .pot file, push the new .pot file into Transifex, wait for new translations (?), pull them here.

WDYT?